README.md

```diff
@@ -46,6 +46,15 @@ redis:
 This is also where you will place any other customizations, such as API tokens
 or certificate paths.
 
+Config settings that vary in value depending on the deployment environment will need
+to be additionally  in the relevant [devops](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api) configs. Things such as:
+
```

Comment:
Maybe mention the three files?  For example:

"...configs (i.e. the `dev-`, `staging-`, and `prod-settings.local.yml.j2` files)."

Reply:
Not everybody has access to that repo. I don't object to having the pointer for those who _do_ have access and in fact would like our developers to take a crack at submitting their own devops PRs for configuration. But maybe add (Private Repo) in the link text or something.

Reply:
"will need to be additionally in" => will also need to be set appropriately for each environment in"

Reply:
Maybe a note about working with platform team to do this part, since there are some non-obvious aspects:
- Internal hosts need to go through forward proxy, so what gets set in settings.local.yml is not the actual hostname, but the forward proxy hostname.
- Credentials must go into credstash and be injected from there at deployment time - don't want people putting a PR with credentials even in a private repo.

Reply:
Thanks @patrickvinograd! I'll get these and @hpjaj's updates in.